tests/unit/manage/test_views.py

```diff
@@ -3916,17 +3897,22 @@ def add_organization_project(*args, **kwargs):
         view = views.ManageOrganizationProjectsViews(organization, db_request)
         result = view.add_organization_project()
 
+        # The project was created, and belongs to the organization.
+        project = (
+            db_request.db.query(Project).filter_by(name="fakepackage").one_or_none()
+        )
+        assert project is not None
+        assert project.organization == organization
+
         assert isinstance(result, HTTPSeeOther)
         assert result.headers["Location"] == db_request.path
-        assert validate_project_name.calls == [pretend.call(project.name, db_request)]
-        assert add_project.calls == [pretend.call(project.name, db_request)]
-        assert len(organization.projects) == 2
```

Comment:
Flagging for review: the asserts I've added should be equivalent to the ones I removed, which stopped working (probably because of stubbing/mocking, but I didn't fully root-cause it).